CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * [ENHANCEMENT] Query-frontend: track query HTTP requests in the Activity Tracker. #3561
 * [BUGFIX] Log the names of services that are not yet running rather than `unsupported value type` when calling `/ready` and some services are not running. #3625
 * [BUGFIX] Alertmanager: Fix template spurious deletion with relative data dir. #3604
+* [BUGFIX] Querier: Remove assertion that the `-querier.max-concurrent` flag must also be set for the query-frontend. #3678
```

Comment:
Move it to the "Documentation" section of the CHANGELOG, please. This just fixes the doc, not the logic.

Reply:
Done.